pkg/network/network.go

```diff
@@ -3,17 +3,26 @@ package network
 import (
 	"fmt"
 	"os"
-	"path/filepath"
 	"sort"
 	"strconv"
 	"strings"
 
 	"github.com/containernetworking/cni/libcni"
 	"github.com/containernetworking/cni/pkg/types"
-	"github.com/sylabs/singularity/internal/pkg/buildcfg"
 	"github.com/sylabs/singularity/internal/pkg/util/env"
 )
 
+type netError string
```

Comment:
Any arguments against using standard `errors.New`? Underneath it is the same, with exception that errors would be `var`, not `const`

Reply:
Yes I want to expose error as constant not variable to avoid modification from another package